var/spack/repos/builtin/packages/libaec/package.py

```diff
@@ -26,16 +26,20 @@ class Libaec(CMakePackage):
     version('1.0.1', sha256='3668eb4ed36724441e488a7aadc197426afef4b1e8bd139af6d3e36023906459')
     version('1.0.0', sha256='849f08b08ddaaffe543d06d0ced5e4ee3e526b13a67c5f422d126b1c9cf1b546')
 
+    variant('static', default=False,
+            description='Builds a static version of the library')
+
     @property
     def libs(self):
         query = self.spec.last_query
+        static = '+static' in self.spec
 
         libraries = ['libaec']
 
         if 'szip' == query.name or 'szip' in query.extra_parameters:
             libraries.insert(0, 'libsz')
 
-        shared = 'static' not in query.extra_parameters
+        shared = 'static' not in query.extra_parameters and not static
```

Comment:
```suggestion
        shared = '~shared' not in self.spec
```

it's double negation so that old builds without the shared variant still look for shared libs.